kratos/tests/cpp_tests/sources/test_checks.cpp

```diff
@@ -35,8 +35,8 @@ namespace Kratos {
 			KRATOS_EXPECT_EQ(1.0, 1.0);
 			KRATOS_EXPECT_NE(1.0, 2.0);
 
-			KRATOS_EXPECT_EQ("Test", "Test");
-			KRATOS_EXPECT_NE("Test ", "Test");
```

Comment:
this change does not seem right, should work either way. Can you elaborate please?

Reply:
In this branch KRATOS_EXPECT_EQ is replaced with EXPECT_EQ which tests memory allocations for C strings. I guess these two lines were used to check KRATOS_EXPECT_* functionality for C strings. Perhaps, it is better to remove them now.